resources/templates/_footer.html

```diff
@@ -56,10 +56,16 @@ <h3 style="color:#FFF;">Public Resources</h3>
                             <a href="http://schedule.csh.rit.edu/" class="sitemap-link">CSH RIT Schedule Maker</a>
                         </li>
                         <li>
-                            <a href="https://github.com/ComputerScienceHouse" class="sitemap-link"><i class="fa fa-github"></i> CSH on GitHub</a>
+                            <a href="https://github.com/ComputerScienceHouse" class="sitemap-link"><i
+                                    class="fa fa-github"></i> CSH on GitHub</a>
                         </li>
                         <li>
-                            <a href="https://twitter.com/CSH_History" class="sitemap-link"><i class="fa fa-twitter"></i> @CSH_History</a>
+                            <a href="https://www.facebook.com/RITCSH/" class="sitemap-link"><i
```

Comment:
Could you not have the line break at 80chars here? IMO it makes reading the HTML source more difficult.